module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -227,6 +227,29 @@ public function getAccess($refresh = false)
         return $result;
     }
 
+    /**
+     * Is Overdrive content active?
+     *
+     * @return bool
+     */
+    public function getContentActive()
```

Comment:
Since this is a boolean method, would this be a better name?

```suggestion
    public function isContentActive()
```